MIVisionX-setup.py

```diff
@@ -195,7 +195,14 @@
         # package dependencies
         os.system('sudo -v')
         os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y ' +
-                  linuxSystemInstall_check+' install libssl-dev libboost-all-dev libboost-dev libboost-system-dev libboost-filesystem-dev')
+                  #linuxSystemInstall_check+' install libssl-dev libboost-all-dev libboost-dev libboost-system-dev libboost-filesystem-dev')
+                  linuxSystemInstall_check+' install libssl-dev')
+        # Boost V 1.74.0 from source
+        os.system('(cd '+deps_dir+'; curl -O -L https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2 )')
+        os.system('(cd '+deps_dir+'; tar xjvf boost_1_74_0.tar.bz2 )')
+        os.system('(cd '+deps_dir+'/boost_1_74_0/; ./bootstrap.sh --prefix=/usr/lib --with-python=python3 )')
+        os.system('(cd '+deps_dir+'/boost_1_74_0/; ./b2 stage -j16 threading=multi )')
+        os.system('(cd '+deps_dir+'/boost_1_74_0/; sudo ./b2 install threading=multi link=static --with-system --with-filesystem)')
```

Comment:
@kiritigowda You need to do a 'sudo ./b2 install threading=multi link=static --with-system --with-filesystem' without the 'link=static' also

Reply:
This only gives the .a files. We need the .so files as well

Reply:
I think if you don't specify anything, it installs both static and shared libs

Reply:
I have resolved the static lib build